src/crypto-api.ts

```diff
@@ -39,6 +39,15 @@ export interface CryptoApi {
      */
     globalBlacklistUnverifiedDevices: boolean;
 
+    /**
+     * Return the current version of the crypto module.
+     * Rust crypto: the format is `Rust SDK ${versions.matrix_sdk_crypto}, Vodozemac ${versions.vodozemac}`
+     * Old Crypto: the format is `Olm x.x.x`
+     * XXXX: remove old crypto when we remove support for it
```

Comment:
Nit: this isn't terribly legible as it stands (see https://pr3825--js-sdk-docs-previews.netlify.app/stable/interfaces/crypto.cryptoapi#getVersion): it could do with some paragraph breaks.

Anyway, I feel like we should just give an example, rather than documenting this in a way that makes it sound like someone can rely on that exact format.

```suggestion
     * Return the current version of the crypto module.
     *
     * For example: `Rust SDK ${versions.matrix_sdk_crypto}, Vodozemac ${versions.vodozemac}`.
     *
```